LICENSE

```diff
@@ -27,7 +27,20 @@ The Chapel implementation is composed of two categories of code:
      vim              vim-based syntax coloring                    VIM
 
    third-party/
-     chpldoc-venv     python packages supporting chpldoc           see third-party/README
+     chpldoc-venv/    python packages supporting chpldoc
```

Comment:
This is a little confusing since there isn't a corresponding third-party/chpldoc-venv/Sphinx directory or file, for example. Maybe remove the trailing slash on chpldoc-venv ?